tutorials/rendering/viewports.rst

```diff
@@ -146,16 +158,32 @@ following code:
    # Set sprite texture.
    sprite.texture = tex
 
+ .. code-tab:: csharp
+
+    // Retrieve the captured Image using get_image().
+    var img = GetViewport().GetTexture().GetImage();
+    // Convert Image to ImageTexture.
+    var tex = ImageTexture.CreateFromImage(img);
+    // Set sprite texture.
+    sprite.Texture = tex;
+
 But if you use this in ``_ready()`` or from the first frame of the :ref:`Viewport's <class_Viewport>` initialization,
 you will get an empty texture because there is nothing to get as texture. You can deal with
 it using (for example):
 
-::
+.. tabs::
+ .. code-tab:: gdscript GDScript
 
-   # Wait until the frame has finished before getting the texture.
+    # Wait until the frame has finished before getting the texture.
    await RenderingServer.frame_post_draw
    # You can get the image after this.
 
+ .. code-tab:: csharp
+
+    // Wait until the frame has finished before getting the texture.
+    RenderingServer.FramePostDraw += HandlePostDraw;
+    // You can get the image after this.
```

Comment:
The closest equivalent to the GDScript code would be:

```csharp
await RenderingServer.Singleton.ToSignal(RenderingServer.SignalName.FramePostDraw);
```

Otherwise, the last comment looks out of place since it should be inside the `HandlePostDraw` method which is not shown here.